app.py

```diff
@@ -326,6 +328,14 @@ def clipCollection(image):
     collection = collection.map(clipCollection)
     return collection
 
+def parse_gpkg_coords_to_ee_geometry(file):
+    # load GPKG file data to fiona collection
+    with fiona.open(fp=file) as f:
```

Comment:
it's trying to open just a name file.gpkg instead of accessing the path of  an actual file written to a temporary file, so you need to get the `tmp.name`. because for now it is crashing the app and fiona throwing a DriverError